packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTable/TransactionsTableCustomAddressLabel.tsx

```diff
@@ -0,0 +1,19 @@
+import { MergedTransaction } from '../../../state/app/state'
```

Comment:
<img width="330" alt="Screenshot 2023-07-19 at 12 43 11" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/47932951/337e7d87-e9b7-4208-bb79-97a529c868b5">

Reply:
The design is outdated. Refer to Figma for the latest.